src/main/java/ubc/pavlab/rdp/model/AccessToken.java

```diff
@@ -23,6 +23,7 @@ public class AccessToken extends Token implements UserContent {
     private Integer id;
 
     @ManyToOne(optional = false)
+    // FIXME: make this token non-nullable, see https://github.com/PavlidisLab/rdp/issues/166
```

Comment:
This is fixed in 2bfa1aea3790ec5b3ae97da7c8c0c02f7ccafab3. This release will have to wait until the 1.4.11 is completed because it will affect order of migration.

Separate concern: we have to freeze the migration process on the 1.4 series once the 1.5 is out.